bin/startup

```diff
@@ -12,5 +12,5 @@ end
 
 chdir APP_ROOT do
   puts "== STARTING UP =="
-  system! "bundle exec foreman start -f Procfile.dev"
+  system! "foreman start -f Procfile.dev"
```

Comment:
This doesnt work for me in the context of this setup file. I approved bc I tested the command by itself and that worked but when I try to run `bin/setup` it fails saying it can't find the executable. Probably me just messing up directories or not having the gem installed properly but figured I would mention it.

Reply:
The related instruction have changed as well: one now must install the `foreman` gem outside of `bundler` via `gem install foreman`. Want to give that a shot @mstruve?

Reply:
Yeah that's what I did, hence the `foreman start -f Procfile.dev` works on its own